easybuild/easyconfigs/d/dammit/dammit-0.3.2-intel-2017a-Python-2.7.13.eb

```diff
@@ -0,0 +1,87 @@
+easyblock = 'Bundle'
+
+name = 'dammit'
+version = '0.3.2'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://www.camillescott.org/dammit/'
+description = """dammit is a simple de novo transcriptome annotator. It was born out of
+ the observations that annotation is mundane and annoying,
+ all the individual pieces of the process exist already, and the existing
+ solutions are overly complicated or rely on crappy non-free software."""
+
+toolchain = {'name': 'intel', 'version': '2017a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.13'),
+    ('X11', '20170314'),
+    ('matplotlib', '2.0.2', '-Python-%(pyver)s-Qt-4.8.7'),
+    ('EMBOSS', '6.6.0', '-X11-20170314'),
+    ('HMMER', '3.1b2'),
+    ('Infernal', '1.1.2'),
+    ('BUSCO', '1.22', '-Python-%(pyver)s'),  # Exact version needed
+    ('LAST', '869'),
+    ('TransDecoder', '2.1.0', '-Perl-5.24.1'),
+    ('khmer', '2.1.1', '-Python-%(pyver)s'),
+    ('crb-blast', '0.6.9'),
+]
+
+exts_list = [
+    ('cloudpickle', '0.4.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/cloudpickle'],
+        'checksums': ['5bb83eb466f0733dbd077e76cf1a15c404a94eb063cecc7049a1482fa1b11661'],
+    }),
+    ('pyinotify', '0.9.6', {
+        'source_urls': ['https://pypi.python.org/packages/source/p/pyinotify'],
+        'checksums': ['9c998a5d7606ca835065cdabc013ae6c66eb9ea76a00a1e3bc6e0cfe2b4f71f4'],
+    }),
+    ('configparser', '3.5.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/c/configparser'],
+        'checksums': ['5308b47021bc2340965c371f0f058cc6971a04502638d4244225c49d80db273a'],
+    }),
+    ('doit', '0.29.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/d/doit'],
+        'checksums': ['fcb479482644de3e83d6faed9b29373f1f38942b79b991a7432943a84416d5cd'],
+    }),
+    ('nose', '1.3.4', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/nose'],
+        'checksums': ['76bc63a4e2d5e5a0df77ca7d18f0f56e2c46cfb62b71103ba92a92c79fab1e03'],
+    }),
+    ('nose-capturestderr', '1.0', {
+        'modulename': 'nose_capturestderr',
+        'source_urls': ['https://pypi.python.org/packages/source/n/nose-capturestderr'],
+        'checksums': ['8a05620bec5acaf006acfe59ad1225673350bba359e1b946ddba53206a3f8c98'],
+    }),
+    ('ficus', '0.3.3', {
+        'source_urls': ['https://pypi.python.org/packages/source/f/ficus'],
+        'checksums': ['7dbd69a4aceb0406e53cf8ede98c039a03110b7b8860b64356554caf537b9324'],
+    }),
+    ('numexpr', '2.6.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/numexpr'],
+        'checksums': ['6ab8ff5c19e7f452966bf5a3220b845cf3244fe0b96544f7f9acedcc2db5c705'],
+    }),
+    (name, version, {
+        'patches': ['dammit-0.3.2_nodocs.patch', 'dammit-0.3.2_py2busco.diff'],
```

Comment:
@hajgato Please rename `dammit-0.3.2_py2busco.diff` to `dammit-0.3.2_py2busco.patch`, and include one patch per line:

```python
 'patches': [
    'dammit-0.3.2_nodocs.patch',
    'dammit-0.3.2_py2busco.patch',
],

Reply:
@boegel: done